database/populateReviews.js

```diff
@@ -0,0 +1,238 @@
+module.exports.reviewOptions = [
+  {
+    posted_date: '2019-10-30',
+    user_name: 'amyG',
+    review_title: 'This thing is great!',
+    review_text: 'Exceeded expectations. It seriously fit our home perfectly',
+    overall_rating: 5,
+    product_value_for_money: 5,
+    product_quality: 5,
+    product_appearance: 5,
+    product_ease_of_assembly: 5,
+    product_works_as_expected: 5,
+    review_helpful: 0,
+    review_helpful_no: 0,
+    product_recommended: 1
+  },
+  {
+    posted_date: '2019-11-25',
+    user_name: 'betty C',
+    review_title: 'I love it!',
+    review_text:
+      'This was perfect for us. Even my mother-in-law complemented me on it(and she usually hates everything I pick)!',
+    overall_rating: 5,
+    product_value_for_money: 4,
+    product_quality: 4,
+    product_appearance: 5,
+    product_ease_of_assembly: 3,
+    product_works_as_expected: 5,
+    review_helpful: 0,
+    review_helpful_no: 0,
+    product_recommended: 1
+  },
+  {
+    posted_date: '2019-12-01',
+    user_name: 'Joe Pumpkin',
+    review_title: 'Nice fit to our home',
+    review_text:
+      'Overall we are satisfied with it. However, I do wish it was easier to assemble. Must have spent three hours putting it together!',
+    overall_rating: 4,
+    product_value_for_money: 4,
+    product_quality: 5,
+    product_appearance: 4,
+    product_ease_of_assembly: 2,
+    product_works_as_expected: 4,
+    review_helpful: 0,
+    review_helpful_no: 0,
+    product_recommended: 1
+  },
+  {
+    posted_date: '2019-10-18',
+    user_name: 'OscarG.',
+    review_title: 'So many complements!',
+    review_text: "SO many people have complemented me on this item! ah! It's great!",
+    overall_rating: 5,
+    product_value_for_money: 5,
+    product_quality: 4,
+    product_appearance: 5,
+    product_ease_of_assembly: 3,
+    product_works_as_expected: 4,
+    review_helpful: 0,
+    review_helpful_no: 0,
+    product_recommended: 1
+  },
+  {
+    posted_date: '2019-05-03',
+    user_name: 'JD',
+    review_title: 'Wearing down already',
+    review_text:
+      'We love the look of it, but the quality is lacking. Unfortunately, after only 6 months it is already showing wear...',
+    overall_rating: 2,
+    product_value_for_money: 2,
+    product_quality: 1,
+    product_appearance: 5,
+    product_ease_of_assembly: 4,
+    product_works_as_expected: 2,
+    review_helpful: 0,
+    review_helpful_no: 0,
+    product_recommended: 0
+  },
+  {
+    posted_date: '2019-10-10',
+    user_name: 'Elsa',
+    review_title: 'Missing parts! ',
+    review_text:
+      'I spent four hours trying to put this together, only to realize I was missing a part! Ugh. I was able to get them, but had to return to the store.',
```

Comment:
did the snow blow white on the mountain that night when she had to go to the store?